### ISSUE TITLE
Oops! UI is crashed! S.forEach is not a function

### ISSUE THREAD

**Issue by Emalton** [morpheus65535/bazarr#1886]
**Describe the bug**
Oops! UI is crashed!
S.forEach is not a function

**To Reproduce**
1. Go to 'A movie'
2. Click on 'History'
3. See error

**Expected behavior**
Not error out

**Screenshots**

**Software (please complete the following information):**
 - Bazarr: [e.g. v 0.6.1] 1.0.4
 - Radarr version [e.g. v 0.2.0.0001] 4.1.0.6175
 - Sonarr version [e.g. v 2.0.0.0001] 3.0.8.1507
 - OS: [e.g. Windows 10] Docker

**Additional context**
```
TypeError: S.forEach is not a function
    at ra (index.c6b646f7.js:807:90579)
    at index.c6b646f7.js:807:134268
    at Object.useMemo (vendors.10e6a3a1.js:30:58852)
    at b.useMemo (vendors.10e6a3a1.js:12:5877)
    at n.useTable (index.c6b646f7.js:807:134176)
    at Ma (index.c6b646f7.js:807:142299)
    at tl (vendors.10e6a3a1.js:30:52929)
    at vd (vendors.10e6a3a1.js:34:8710)
    at hd (vendors.10e6a3a1.js:34:940)
    at pm (vendors.10e6a3a1.js:34:871)
Ws @ vendors.10e6a3a1.js:32

```

Additional info on request, TIA!

**Comment by morpheus65535** [comment]
Probably fixed in 1.0.5-beta. Can you test it?

**Comment by Emalton** [comment]
Yes, it works great! Thank you!
